docs/source/quantization/bnb.rst

```diff
@@ -0,0 +1,41 @@
+.. _bits_and_bytes:
+
+BitsAndBytes
+==================
+
+vLLM now supports `BitsAndBytes <https://github.com/TimDettmers/bitsandbytes>`_ for more efficient model inference.
+BitsAndBytes quantizes models to reduce memory usage and enhance performance without significantly sacrificing accuracy.
+This is particularly useful for deploying large language models in resource-constrained environments.
+Below are the steps to utilize BitsAndBytes with vLLM.
+
+.. code-block:: console
+
+    $ pip install bitsandbytes>=0.42.0
+
+vLLM reads the model's config file and supports both in-flight quantization and pre-quantized checkpoint.
+
+Read quantized checkpoint
+--------------------------
+
```

Comment:
Please add a line on how to find checkpoints that satisfy this criteria i.e. the "bitsandbytes" tag seems to be close to this https://huggingface.co/models?other=bitsandbytes